src/lib/constants.ts

```diff
@@ -0,0 +1,47 @@
+import { checkString } from './validation';
+
+const bootMountPointFromEnv = checkString(process.env.BOOT_MOUNTPOINT);
+const rootMountPoint = checkString(process.env.ROOT_MOUNTPOINT) || '/mnt/root';
+
+const supervisorNetworkInterface = 'supervisor0';
+
+export = {
+	rootMountPoint,
+	databasePath: checkString(process.env.DATABASE_PATH) || '/data/database.sqlite',
+	dockerSocket: process.env.DOCKER_SOCKET || '/var/run/docker.sock',
+	supervisorImage: checkString(process.env.SUPERVISOR_IMAGE) || 'resin/rpi-supervisor',
+	ledFile: checkString(process.env.LED_FILE) || '/sys/class/leds/led0/brightness',
+	// Only used in development
+	forceSecret: {
```

Comment:
This can be removed (it stopped being used with #482)

Reply:
Sure, will do it in this PR. In general I'm trying to change as little as possible, but given this is an easy win I can do that.